utils.py

```diff
@@ -73,6 +73,11 @@ def get_cbl_name():
         if "archlinux" in base_config:
             return "x86_64"
 
+    # ChromeOS configurations have the architecture as the second to last
+    # folder of the second config fragment path
```

Comment:
Can you put an example in the comment? That makes it easier to review what this is doing.

Reply:
Done.